core/modules/views/tests/handlers/views_handler_field_bulk_form.test

```diff
@@ -0,0 +1,247 @@
+<?php
+
+/**
+ * @file
+ * Definition of ViewsHandlerFieldBulkFormTest.
+ */
+
+require_once BACKDROP_ROOT . '/core/modules/views/tests/views_query.test';
```

Comment:
As views_handler_field_bulk_form.test ships with the same module, another option for that require_once would be `__DIR__`.

Reply:
The issue is that `views_query.test` is not in the same directory as `views_handler_field_bulk_form.test`, but the one above it.  This exact same require_once line is used in many another existing views test, so I prefer not to mess up with it.